### DISCUSSION TITLE
Connection cannot be established

### DISCUSSION THREAD

**Discussion by RedSpid3r** [LeonMarcel-HD/StreamDeck-TeamSpeak5#9] [Q&A / Help] [answered]
**Describe the bug**

**To Reproduce**
1. Install streamdeck plugin
2. Attempt to use Mute Microphone or Mute Output

**Expected behavior**
Microphone or Output is muted

**Screenshots**

**Additional context**
 - Operating System: Windows 11
 - Stream Deck Version:  6.4.1.19697
 - Stream Deck Hardware: Stream Deck MK1
 - Plugin Version: 1.0.0
 - TeamSpeak Version: Version: 5.0.0-beta76

**Comment by LeonMarcel-HD** [comment]
Did you accept the request in TeamSpeak?
Did you follow the setup instructions on how to bind the Button to TeamSpeak?

**Comment by RedSpid3r** [comment, marked as answer]
> Did you accept the request in TeamSpeak? Did you follow the setup instructions on how to bind the Button to TeamSpeak?

Yes to both, however when I opened Options -> Remote Apps "Stream Deck Plugin" was listed for a split second and then disappeared.

So, I completely removed the plugin from StreamDeck, disabled then enabled Remote Apps in TeamSpeak then reinstalled the plugin and re-bound the keys and it seems to be working now. I can only assume something bugged out during the flow of accepting the request in TS and it didn't go through properly.

You can go ahead and close this out now. Thanks for the help!

**Comment by LeonMarcel-HD** [comment]
For some reason the plugin list inside of TeamSpeak is not working correctly/only showing after a restart the plugins.
